src/containers/TriggerTemplate/TriggerTemplate.js

```diff
@@ -0,0 +1,386 @@
+    } = prevProps;
+    const {
+      namespace: prevNamespace,
+      triggerTemplateName: prevTriggerTemplateName
+    } = prevMatch.params;
+
+    if (
+      namespace !== prevNamespace ||
+      triggerTemplateName !== prevTriggerTemplateName ||
+      (webSocketConnected && prevWebSocketConnected === false)
+    ) {
+      this.fetchData();
+    }
+  }
+
+  fetchData() {
+    const { match } = this.props;
+    const { namespace, triggerTemplateName } = match.params;
+    this.props.fetchTriggerTemplate({ name: triggerTemplateName, namespace });
+  }
+
+  render() {
+    const {
+      intl,
+      error,
+      match,
+      selectedNamespace,
+      triggerTemplate
+    } = this.props;
+
+    if (error) {
+      return TriggerTemplateContainer.notification({
+        kind: 'error',
+        intl
+      });
+    }
+    if (!triggerTemplate) {
+      return TriggerTemplateContainer.notification({
+        kind: 'error',
+        message: 'TriggerTemplate not available',
+        intl
+      });
+    }
+
+    const { params, resourcetemplates } = triggerTemplate.spec;
+    const { triggerTemplateName } = match.params;
+    const triggerTemplateDate = triggerTemplate.metadata.creationTimestamp;
+    const triggerTemplateNamespace = triggerTemplate.metadata.namespace;
+
+    /* 
+      Labels: ${"baz":"boo","foo":"bar"} 
+      should become 
+      Labels:
+       foo: bar
+       baz: boo
+    */
+
+    let triggerTemplateLabels = '';
+    const formattedLabelsToRender = [];
+    if (triggerTemplate.metadata.labels) {
+      triggerTemplateLabels = JSON.stringify(triggerTemplate.metadata.labels)
+        .replace('{', '')
+        .replace('}', '')
+        .replace(/['"]+/g, '')
+        .replace('$', '');
+
+      const templateLabelsSplitOnComma = triggerTemplateLabels.split(',');
+      templateLabelsSplitOnComma.forEach(item => {
+        const splitOnColon = item.split(':');
+        formattedLabelsToRender.push(`  ${splitOnColon[0]}: ${splitOnColon[1]}
+`);
+      });
+    }
+
+    const headersForParameters = [
+      {
+        key: 'name',
+        header: intl.formatMessage({
+          id: 'dashboard.triggerTemplate.name',
+          defaultMessage: 'Name'
+        })
+      },
+      {
+        key: 'default',
+        header: intl.formatMessage({
+          id: 'dashboard.triggerTemplate.default',
+          defaultMessage: 'Default'
+        })
+      },
+      {
+        key: 'description',
+        header: intl.formatMessage({
+          id: 'dashboard.triggerTemplate.description',
+          defaultMessage: 'Description'
+        })
+      }
+    ];
+
+    let rowsForParameters = [];
+
+    if (params) {
+      rowsForParameters = params.map(
+        ({ name, default: defaultValue, description }) => ({
+          id: name,
+          name,
+          default: defaultValue,
+          description
+        })
+      );
+    }
+    return (
+      <>
+        <h1>{triggerTemplateName}</h1>
+        <Tabs>
```

Comment:
You are using the Tabs component in a different manner than I did in the TriggerBinding page. In your approach you don't even use the state variables which can be omitted. I feel we should be using the same approach so @AlanGreene let me know which one of use has to compromise. Also I see you are using `ghost` tabs were I am not so we should also come to the agreement on what kind it should be.

Reply:
![image](https://user-images.githubusercontent.com/12814972/70341667-612a5700-184b-11ea-8ea6-404ccd3a3837.png)

Think I'm good 😄

Reply:
![image](https://user-images.githubusercontent.com/12814972/70341733-861eca00-184b-11ea-9cd1-e89068f28d84.png)